Makefile

```diff
@@ -83,6 +83,16 @@ run-functional-tests: testnnp test-registry
 testnnp:
 	cd misc/testnnp; $(MAKE) $(MFLAGS)
 
+pause-container:
+	@docker build -f scripts/dockerfiles/Dockerfile.buildPause -t "amazon/amazon-ecs-build-pause-bin:make" .
+	@docker run --net=none -e TARGET_OS="${TARGET_OS}" \
```

Comment:
I'm not sure if we actually need `TARGET_OS` here; we're not building a pause container for Windows (yet?).